src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -208,74 +221,101 @@ report_broker_config(describe_configs_result& result, bool include_synonyms) {
         }
     }
 
-    add_broker_config(
-      result,
-      "listeners",
-      config::node().kafka_api,
-      include_synonyms,
-      &kafka_endpoint_format);
+    ss::sstring property_name = "listeners";
```

Comment:
can this be a `std::string_view` ?

Reply:
Yes it can, but why is it better?
Previosly the same thing in this file was made with `ss::sstring`

Reply:
now every time we assign string we copy it.

Reply:
I will remove it in next iteration, return to old way passing strings.